canonical-web-and-design/cn.ubuntu.com
# Add newsletter form to blog

## Done

Add newsletter form to blog homepage and blog post page

## QA

- Go to /blog
- Check that there is a newsletter subscription form
- Check that the form works
- Go to a blog post page
- Check that there is a newsletter subscription form
- Check that the form works
- Check that the form copy matches [the copy doc](https://docs.google.com/document/d/12FTdNsmhTF6KPtYN5lQ8iFQgvU9BgJDdCvUjiScc-Ok/edit)

## Issue / Card

Fixes #414
